test/shutdown-testing.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -eu
+
+main(){
+  echo "Stopping spring app initialized..."
+
+  local appPID="$(jps -l | grep edu.wgu.osmt.ApplicationKt | awk '{print $1}')"
+  echo "Got app PID ${appPID}"
+
+  kill -SIGINT ${appPID}
+  echo "Spring app has been stopped"
```

Comment:
Either put the kill command & message in a do-while loop to repoll the PID to check whether the app has actually stopped, or atleast have the script wait one time and then recheck the PID then provide a message. Loop pseudo-example:
```
retry_limit = some value
retry_count = 0

do {
echo "stopping spring app"
kill app
wait x // need to make sure this is non-blocking
} while (spring app running && retry_count++ < retry_limit)

if (app stoppped) { echo "app has stopped" }
else { app failed to shutdown }
```